ml-fundamentals/ml-experimentation.md

```diff
@@ -0,0 +1,147 @@
+# Model Experimentation
+
+Machine learning model experimentation involves uncertainty around the expected model results and future operationalization.
+To handle this uncertainty as much as possible, we propose a semi-structured process, balancing between engineering/research best practices and rapid model/data exploration.
+
+## Model experimentation goals
+
+1. **Performance**: Find the best performing solution
+2. **Operationalization**: Keep an eye towards production, making sure the operationalization is feasible
+3. **Code quality** Maintain code and artifacts quality
+4. **Reproducibility**: Keep research active by allowing experiment tracking and reproducibility
+5. **Collaboration**: Foster the collaboration and joint work of multiple people on the team
+
+## Model experimentation challenges
+
+- **Trial and error process**: Difficult to plan and estimate durations and capacity.
```

Comment:
Nit. The section above is bulleted, this section is numbered. Should be consistent. Also, the first bullet ends with ".", the others don't. Not sure which is right, but should be consistent.